boot_node/src/server.rs

```diff
@@ -44,7 +44,8 @@ pub async fn run<T: EthSpec>(config: BootNodeConfig<T>, log: slog::Logger) {
     info!(log, "Contact information"; "multiaddrs" => ?local_enr.multiaddr_p2p());
 
     // construct the discv5 server
-    let mut discv5 = Discv5::new(local_enr.clone(), local_key, discv5_config).unwrap();
+    let mut discv5: Discv5<lighthouse_network::discv5::DefaultProtocolId> =
```

Comment:
Here we can take advantage of the default
```suggestion
    let mut discv5: Discv5 =
```